README.rst

```diff
@@ -169,12 +169,12 @@ Developers". Feel free to cite our work in your research using the following Bib
                       Quinn, Jamie and
                       Wessel, Paul},
       title        = {{PyGMT: A Python interface for the Generic Mapping Tools}},
-      month        = march,
+      month        = april,
```

Comment:
I'm not sure if `april` works, but based on this page (https://www.bibtex.com/f/month-field/), we should use either `apr` or `April`
```suggestion
      month        = apr,
```

Reply:
Should use `apr` I think, at least my Zotero bibliography manager uses the abbreviated form.